### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by derrick-fleming**
https://user-images.githubusercontent.com/109825222/189953430-09c2b5d1-49d0-4808-96a7-e62b56287619.mov

Screen Shot 2022-09-13 at 9 06 46 AM

https://user-images.githubusercontent.com/109825222/189953579-a40ed650-60ee-41a7-9b0b-8f35791c4c23.mov

https://user-images.githubusercontent.com/109825222/189953655-26b68fc2-7f42-4cbd-bdfa-a081c4029a30.mov

https://user-images.githubusercontent.com/109825222/189954336-53a198bd-0418-4f96-b1d1-b0c7b68c75f2.mov

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,11 +14,36 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Proza+Libre:wght@400;500;600&display=swap" rel="stylesheet">
 </head>
 <body>
   <header>
+    <div class="container row">
```
Comment (codingcodymiller) [inline index.html]: One element should not be both a `row` and a `container`. You should make this element the `container` and give it a child with the `row` class.

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  event.preventDefault();
+  var formInputValues = {
+    title: $codeJournal.elements.title.value,
+    photoUrl: $codeJournal.elements.photoUrl.value,
+    textArea: $codeJournal.elements.notes.value,
+    entryNumber: dataModel.nextEntryId
+  };
+  dataModel.nextEntryId++;
+
+  dataModel.entries.push(formInputValues);
```
Comment (codingcodymiller) [inline js/main.js]: You want to add to the beginning, not the end.

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+  $placeHolderImage.setAttribute('src', 'images/placeholder-image-square.jpg');
+  $codeJournal.elements.title.value = '';
+  $codeJournal.elements.photoUrl.value = '';
+  $codeJournal.elements.notes.value = '';
```
Comment (codingcodymiller) [inline js/main.js]: You should use the `reset` method on the `form` element instead 🤓

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+if (previousData !== 'null') {
+  dataModel = JSON.parse(previousData);
+}
+
+window.addEventListener('beforeunload', storeToLocalStorage);
+
+function storeToLocalStorage(event) {
+  var dataJSON = JSON.stringify(dataModel);
+  localStorage.setItem('code-journal-storage', dataJSON);
+}
```
Comment (codingcodymiller) [inline js/main.js]: Please relocate this code to `data.js` so that it is in the same file as the variable it is preserving.

File: js/main.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+var $codeJournal = document.querySelector('#code-journal');
+var $placeHolderImage = document.querySelector('.placeholder-image');
+var $photoUrlInput = document.querySelector("input[name='photoUrl']");
+var dataModel = data;
```
Comment (codingcodymiller) [inline js/main.js]: I'm not sure what the purpose is of this, I would just stick to using the `data` variable.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by derrick-fleming** [comment]
Screen Shot 2022-09-13 at 9 53 59 AM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**